test/remote/instance.go

```diff
@@ -235,7 +235,7 @@ func (i *InstanceInfo) CreateOrGetInstance(localSSDCount int) error {
 		}
 
 		if i.cfg.CloudtopHost {
```

Comment:
This seems to no longer be necessary on cloudtop, and was actively causing my local test run to fail.